src/qs8-vadd/rvv.c.in

```diff
@@ -0,0 +1,71 @@
+// Copyright 2023 Google LLC
+//
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree.
+
+#include <assert.h>
+#include <riscv_vector.h>
+#include <xnnpack/vbinary.h>
+$assert DATATYPE in ["QS8", "QU8"]
```

Comment:
Dont put effort into QU8 for new microkernels.  We are deprecating it.
QS8, F16, F32 are the supported types, with QD8 for GEMM supporting QC4W for weights.

Reply:
ok